### ISSUE TITLE
Skin causes crash while bootup

### ISSUE THREAD

**Issue by Lolihunter1337** [ppy/osu#4096]
**Describe your problem:** I imported a skin and selected it, while selecting the game crashes and now it wont boot into the menu anymore

**Screenshots or videos showing encountered issue:** https://streamable.com/k63ee

**osu!lazer version:** 2019.117.0

**Logs:** 
[database.log](https://github.com/ppy/osu/files/2775910/database.log)
[network.log](https://github.com/ppy/osu/files/2775911/network.log)
[performance.log](https://github.com/ppy/osu/files/2775912/performance.log)
[runtime.log](https://github.com/ppy/osu/files/2775913/runtime.log)

**Computer Specifications:** i3 6100, 8GB RAM, Windows 10 Pro x64

**Comment by peppy** [comment]
Looks like the skin having a catch section is causing a crash (first error report).

**Comment by peppy** [comment]
@Lolihunter1337 could you please debug and see where the actual crash is happening? The error that is showing up in runtime.log should not cause a catastrophic failure.

**Comment by Lolihunter1337** [comment]
It gives me a stack overflow

**Comment by peppy** [comment]
Can you include the call stack? A screenshot? Anything?

**Comment by Lolihunter1337** [comment]
Not really. "Your app has been paused but there is no code to display because all threads have run external code (usually system or framework code)"
i dont have the framework stuff insalled. if i get the steps to get more info out it then i would be glad to do so

**Comment by peppy** [comment]
If it's a stack overslow you should be able to see where it's overflowing in the call stack window.

**Comment by Lolihunter1337** [comment]
![image](https://user-images.githubusercontent.com/17437302/51434646-6c41bc00-1c66-11e9-81f1-80e2820bce7a.png)


System.StackOverflowException
  HResult=0x800703E9
  Quelle = <Die Ausnahmequelle kann nicht ausgewertet werden.>
  Stapelüberwachung:
<Die Ausnahmestapelüberwachung kann nicht ausgewertet werden.>

**Comment by Lolihunter1337** [comment]
well @peppy 
![image](https://user-images.githubusercontent.com/17437302/51434775-d2c7d980-1c68-11e9-9ca0-db6bfea74cd4.png) thats all i get

**Comment by peppy** [comment]
Aha,

The skin has a texture over 4096px. This is not supported by lazer (and many GPUs).

**Comment by Dragicafit** [comment]
It's not the same issue so I created a new one #4098.
